exploratory-tests/tesztterv.md

```diff
@@ -0,0 +1,126 @@
+Test plan
+
+Unless explicitly written you can reproduce these tests with emulator or real, physical device.
+
+## 1. test
+Test title: Google sign-in testing.\
+Test objective: Testing wether the user is able to log in with his/her Google account.\
+Test steps: 
+- Start the application
+- Click-on Google Sign-in in the middle of the screen.
+Test result: Successful
+
+## 2. test
+Test title: Tracking service testing between 2 near locations.\
+Test objective: Testing wether we can start the service and the service sends location information accordingly\
+Test steps:
+- Log-in with Google-account
```

Comment:
Ha igeként használjuk, akkor log in -nak kell írni. (Ha főnévként használjuk, akkor viszont helyes az egybeírás és a kötőjeles is elvileg.)